packages/larva-css/src/02-algorithms/a-buy-now/a-buy-now.common.inline.scss

```diff
@@ -0,0 +1,103 @@
+@import 'setup';
+
+.lrv-a-buy-now {
```

Comment:
Algorithms should be named according to presentation, not to a component - a lot of this CSS can be accomplished with utilities, for example, the font-related styling should correspond to a type style available in a-font.

Reply:
Style updated to use __ and --

Reply:
I tried using a-font, but ran into a few issues. Would I need to define something new in a-font in order to complete this?